src/main/java/com/zufar/onlinestore/payment/repository/PaymentRepository.java

```diff
@@ -4,4 +4,7 @@
 import org.springframework.data.repository.CrudRepository;
 
 public interface PaymentRepository extends CrudRepository<Payment, Long> {
+
+    Payment findByPaymentIntentId(String paymentIntentId);
+
```

Comment:
remove extra space

Reply:
Done

Reply:
fix it please
there is still an extra space